packages/nextjs-kit/src/components/previewRibbon.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+// import Image from 'next/image';
+
+interface PreviewRibbonProps {
+	clearPreviewRoute?: string;
+}
+
+const PreviewRibbon: React.FC<PreviewRibbonProps> = ({
+	clearPreviewRoute = '/api/clear-preview',
+}) => {
+	const [show, setShow] = useState(true);
+
+	return (
+		<div>
+			<div
+				className={`ps-flex ps-justify-between ps-items-center ps-min-h-[3.5rem] ps-h-14 ps-px-8 ps-py-2 ps-bg-[#3017A1] ps-text-white ps-transition-all ps-duration-300 ps-w-full ${
+					!show
+						? 'ps-justify-start ps-translate-x-[calc(100%-5rem)] ps-pl-4'
+						: ''
+				}`}
+			>
+				{show && (
+					<>
+						<span className="ps-mr-auto ps-self-center">
+							Preview Mode Enabled
+						</span>
+						<Link href={clearPreviewRoute}>
+							<a className="ps-justify-self-end ps-border ps-text-black ps-border-black ps-w-fit ps-max-w- ps-px-4 ps-py-2 ps-mr-12 ps-bg-yellow-300">
```

Comment:
looks like `ps-max-w-` is a leftover from the previous implementation and it can be removed. Unless I'm missing something :)